book_management/api/tests.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.test import TestCase
+from rest_framework.test import APITestCase
+
+from models import Category, Book
+
+class CategoryTest(APITestCase):
+
+    def test_string_representation_of_category(self):
+        category = Category.objects.create(name='Sample category')
+        self.assertEqual(str(category), "<Category name='Sample category', parent_category='None'>")
+
+    def test_can_create_category_without_parent(self):
+        book_data = {
+          'name': 'Science'
+        }
+        response = self.client.post('/api/categories/', book_data)
```

Comment:
Avoid hard coding URLs in tests and use  https://docs.djangoproject.com/en/1.11/ref/urlresolvers/#reverse  instead. That way when the URL changes there would be no need to update the tests